File: android/src/main/java/com/danielisho/app_preferences_flutter/PreferencesFragment.java
package com.danielisho.app_preferences_flutter;

import com.takisoft.preferencex.PreferenceFragmentCompat;
import android.os.Bundle;
import androidx.annotation.Nullable;

public class PreferencesFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferencesFix(@Nullable Bundle savedInstanceState, String rootKey) {
        setPreferencesFromResource(R.xml.preferences, rootKey);
	
    }
}

File: android/src/main/java/com/danielisho/app_preferences_flutter/PreferencesFragment.kt
package com.danielisho.app_preferences_flutter;

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat


class PreferencesFragment : PreferenceFragmentCompat() {
    override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
        setPreferencesFromResource(R.xml.preferences,rootKey)
    }
}